package.json

```diff
@@ -18,6 +18,7 @@
     "lint": "lerna run lint --no-bail",
     "test:unit": "lerna run test:unit --no-bail",
     "test:e2e": "lerna run test:e2e --no-bail",
+    "test:e2e:long": "lerna run test:e2e:long --no-bail",
```

Comment:
maybe call it "sim" (simulation) instead of "long" since long is relative in e2e :smile:

Reply:
A much better name yes! Thanks